src/appleseed/renderer/modeling/object/meshobject.cpp

```diff
@@ -37,6 +37,7 @@
 
```

Comment:
Alpha maps don't make sense for some kind of objects, for example curves.
I prefer to enable them only for objects that make sense.